WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+FancyAlerts.swift

```diff
@@ -81,7 +81,16 @@ extension BlogDetailsViewController {
     }
 
     @objc func shouldShowQuickStartChecklist() -> Bool {
-        return QuickStartTourGuide.shouldShowChecklist(for: blog) && !shouldShowDashboard()
+        if shouldShowDashboard() {
```

Comment:
[nit] `shouldShowDashboard` sounds like if the dashboard should be shown, which is not true. `dashboardEnabled` might be more accurate?

Reply:
Agreed!